### PR TITLE
Add EPSG:31254 MGI / Austria GK West

### PR DESCRIPTION
**Pull request by asaunier**
Source definition and extent available at http://pacificprojections.spc.int/31254
```
proj4.defs("EPSG:31254","+proj=tmerc +lat_0=0 +lon_0=10.33333333333333 +k=1 +x_0=0 +y_0=-5000000 +ellps=bessel +towgs84=577.326,90.129,463.919,5.137,1.474,5.297,2.4232 +units=m +no_defs");
```

### CODE REVIEW

File: src/tmerc.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+/**
+ * @module ngeo.proj.tmerc
+ */
```
Comment (gberaudo) [inline src/tmerc.js]: This comment is not needed.

Comment (asaunier) [reply src/tmerc.js]: @gberaudo I did as in https://github.com/geoblocks/proj/blob/master/src/somerc.js#L2
But will remove it if you think it is not relevant.

Comment (gberaudo) [reply src/tmerc.js]: It is especially the ngeo prefix which makes no sense here.
You can keep the `@module`, it does not hurt but I think it is only needed by jsdoc, which we do not use.

Comment (gberaudo) [reply src/tmerc.js]: If you decide to remove completely then it would be great to remove usage in the other places.

**Pull request merged**